sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Security/Ucfg/UniversalControlFlowGraphBuilder.cs

```diff
@@ -310,22 +326,26 @@ private Expression BuildAssignment(AssignmentExpressionSyntax assignment)
             {
                 var left = GetSymbol(assignment.Left);
 
+                var argument = BuildInstruction(assignment.Right);
```

Comment:
I probably missed something on my previous reviews but you have a `BuildInstruction` method returning an  `argument` variable that seems quite weird to me.

Besides a few lines after there is a `CreateInstruction` method. The name really doesn't help to understand which method is doing what. I kind of expect `Build` and `Create` to be sort of the same.

Reply:
will it be better if `argument` is `right`?